### ISSUE TITLE
keyboard contrast

### ISSUE THREAD

**Issue by kimpham54** [dulibrarytech/digitaldu-frontend#197]
from @metageeky: Contrast on keyboard focus is also lacking. That's a new WCAG 2.1 rule

**Comment by kimpham54** [comment]
keyboard navigation - make tabbing obvious
sometimes esp in video object pages it's hard to tell where you are on the page when you're tabbing - keyboard

**Comment by kimpham54** [comment]
right now light grey. find a more contrasty colour

**Comment by jrynhart** [comment]
Found 3 unnecessary tabstops, will remove from UV dynamically, to prevent the tab stops
Static code can not be updated, the elements with the tabstops are generated by uv script at runtime
1. Remove anchor to "/" of application, first child of .uv-seagragon-extension div
2. Update tabindex value to '-1' on .openseadragon-canvas div
3. Update tabindex value to '-1' on .uv-seagragon-extension div
